lib/validator/v3/validator.go

```diff
@@ -230,6 +232,12 @@ func validateInterface(v *validator.Validate, topStruct reflect.Value, currentSt
 	return s == "*" || interfaceRegex.MatchString(s)
 }
 
+func validateNATParam(v *validator.Validate, topStruct reflect.Value, currentStructOrField reflect.Value, field reflect.Value, fieldType reflect.Type, fieldKind reflect.Kind, param string) bool {
+	s := field.String()
+	log.Debugf("Validate NAT Parameter: %s", s)
+	return s == "*" || natParamRegex.MatchString(s)
```

Comment:
I don't think `*` should be valid here.  `*` is a special-case value for the interface name of a host endpoint so we don't want to copy that here.

Reply:
ack